config.yml

```diff
@@ -160,3 +160,7 @@ default:
                 hg38-nci: "04-24937N-Schmitz"
                 hg38: "BLGSP-71-06-00286-99A-01D"
                 hg19-clc: "PA011-G"
+
```

Comment:
Can we also use this to specify a default metadata for example data sets bundled with the package?

Reply:
Yes, I was thinking of it the same way as well - and if the concept in this draft is something we will move forward with I will switch the metadata (+ ssm, SV, CNV) to the helper package as well

Reply:
I have also omitted from this PR bundling the example metadata and datasets - I think this can be it;s own dedicated PR since we would also want to generate examples for each function using that example data and for metadata we also need to work out minimal required columns